packages/relay-runtime/network/RelayObservable.js

```diff
@@ -477,6 +494,19 @@ function fromValue<T>(value: T): RelayObservable<T> {
   });
 }
 
+function fromAsyncIterator<T>(obj: AsyncIterator<T>): RelayObservable<T> {
+  return RelayObservable.create(sink => {
+    (async () => {
+      while(true) {
+        const {value, done} = await obj.next();
+        if (done) break;
+        sink.next(value);
+      }
```

Comment:
this can alternately be:

```suggestion
    (async () => {
      for await (const patch of obj) {
        sink.next(patch);
      }
      sink.complete();
    })().catch(sink.error);
```

if babel wasnt getting unhappy.